haas/plugins/result_handler.py

```diff
@@ -13,6 +13,14 @@
 from .i_result_handler_plugin import IResultHandlerPlugin
 
 
+def get_test_description(test, descriptions=True):
+    doc_first_line = test.shortDescription()
+    if descriptions and doc_first_line:
+        return '\n'.join((str(test), doc_first_line))
```

Comment:
I'll need to re-confirm behaviour with `unittest`, but in this PR the code has just been moved unchanged from a method to a function to be re-used elsewhere.